Sources/FLCharts/FLCartesianPlane.swift

```diff
@@ -247,10 +252,13 @@ public class FLCartesianPlane: UIView, FLStylable {
             axesLines.addLines(between: [chartTopLeft,
                                          chartBottomLeft,
                                          chartBottomRight])
-        } else {
+        } else if yAxisPosition == .right {
             axesLines.addLines(between: [chartTopRight,
                                          chartBottomRight,
                                          chartBottomLeft])
+        } else {
+            axesLines.addLines(between: [chartBottomLeft,
+                                         chartBottomRight])
```

Comment:
@SvenMuc Can you change this to a switch statement please?

Reply:
Fixed it.